articles/quickstart/native/ios-swift/interactive.md

```diff
@@ -214,9 +214,9 @@ Verify that you can access the `email`, `picture`, or any other [claim](/secure/
 :::
 
 :::checkpoint-failure
-Sorry about that. Here's a couple things to double check:
-- Did you import the `JWTDecode` module in the file where you're calling the `decode(jwt:)` method?
-- Make sure the claims are spelled correctly.
+If your application did not launch successfully:
```

Comment:
```suggestion
If the you cannot access the user information:
```

Reply:
This one should be "If you cannot". Fixed it in https://github.com/auth0/docs/pull/10102/commits/7eaacaad6319e262f263eec8aaea01f126dab80c